cli/src/commands/deploy.ts

```diff
@@ -172,14 +187,24 @@ export default class Deploy extends Command {
     return value === NEW_APPLICATION ? undefined : value;
   }
 
+  public async findApplicationIdByName(appName: string): Promise<string> {
```

Comment:
CR: Use `const { id } = this.lycanClient.getAppByName(appName)`

Reply:
Thanks!